hyperspy/io_plugins/hdf5.py

```diff
@@ -342,7 +391,8 @@ def parse_structure(key, group, value, _type, compression):
         elif isinstance(value, np.ndarray):
             group.create_dataset(key,
                                  data=value,
-                                 compression=compression)
+                                 compression=compression,
+                                 chunks=True)
```

Comment:
My bad, I mixed up the code. I'll remove it, as this branch only deals with reading, and that is writing option/routine